hyperion-geiger-counter/src/main/java/com/willowtreeapps/hyperion/geigercounter/GeigerCounterModule.java

```diff
@@ -56,7 +59,23 @@ protected void onDestroy() {
 
     @Override
     public void onClick(View v) {
-        detector.setEnabled(!detector.isEnabled());
+        boolean isEnabled = detector.isEnabled();
+
+        // When enabling while the speaker is muted, remind the user to unmute.
+        if (!isEnabled) {
+            AudioManager audioManager = (AudioManager) getContext().getSystemService(Context.AUDIO_SERVICE);
+            switch (audioManager.getRingerMode()) {
+                case AudioManager.RINGER_MODE_SILENT:
+                case AudioManager.RINGER_MODE_VIBRATE:
+                    String message = "Please turn up the sound volume.";
+                    Toast.makeText(getContext(), message, Toast.LENGTH_SHORT).show();
```

Comment:
Prefer to pass a String resource identifier as the second argument

Reply:
Should we also check against https://developer.android.com/reference/android/media/AudioManager.html#getStreamVolume(int) ?

Reply:
In my limited tests, I found that the standard volume keys affected the ringer mode when they went to zero volume. I think this stream volume query sounds like a good thing to try if the behavior is different when headphones are in.